src/diffusers/loaders/single_file_utils.py

```diff
@@ -267,7 +270,6 @@
     "cond_stage_model.transformer.",
     "conditioner.embedders.0.transformer.",
 ]
-OPEN_CLIP_PREFIX = "conditioner.embedders.0.model."
```

Comment:
did we just happen to find this line not needed (i.e. this is  not relevant to the fix)?

Reply:
Yeah it was left over and not used anywhere.